packages/manager/apps/web/client/app/hosting/database/order/components/private-database/translations/Messages_fr_FR.json

```
@@ -0,0 +1,30 @@
+{
+  "hosting_database_order_private_title": "Commander un SQL Privé",
+  "hosting_database_order_private_description": "Vous devez avoir un hébergement pour commander les options SQL Privé et Start SQL.",
+  "hosting_database_order_private_step_header_type": "Type de base de données",
+  "hosting_database_order_private_step_header_customize": "Personnalisation",
+  "hosting_database_order_private_step_header_duration": "Choisissez la durée",
```

to be consistent with the rest, i think we should use:

```suggestion
  "hosting_database_order_private_step_header_duration": "Sélectionner la durée",
```

---

This text was already there. I just moved from once place to another to reuse. Are you sure the existing text is wrong?

---

Since this ticket received a validation, I'll merge this Pull Request.
Feel free to open a new Pull Request @mohammed-zahaf regarding your comment.
Thank you!